### ISSUE TITLE
Dragging outside of a div causes drag to stop

### ISSUE THREAD

**Issue by chrisbartley** [CMU-CREATE-Lab/experimental-fast-timeline#7]
Dragging on an axis or the plot area stops if you move the mouse outside of the div.  Probably related to #6

**Comment by pdille** [comment]
Should be fixed in commit 9ec73b7

Note that this solves the issue of dragging from axis to plot area and vice versa.

**Comment by dognotdog** [comment]
@pdille seems like dragging outside the div still stops the update, though it resumes when you enter the div again?

**Comment by pdille** [comment]
@dognotdog Yeah, the changes I made only ensure that you could drag between all the elements _within_ the grapher (hence my comment above). Before if you were dragging from the time axis to the plot or vice-versa, you would lose the ability to drag. That was the most annoying. If we want to also be able to keep dragging once _outside_ the grapher itself, that'll take some further refactoring since all events are currently tied only to grapher related elements. 

Perhaps this issue was originally about dragging outside the grapher, but the more common case I was running into was dragging from axis to plot area and vice versa, so I addressed that.
